math/abella/Portfile

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           ocaml 1.1
+
+name                abella
+version             2.0.7
+revision            0
+categories          math ocaml
+maintainers         {@barracuda156 gmail.com:vital.had} {pmetzger @pmetzger} openmaintainer
```

Comment:
Not sure I can commit to co-maintaining, FWIW. My time is only sporadically available atm.

Reply:
Got it, no issues at all.